### PR TITLE
Feature lwdev 4300 enhance rabbit mq monitoring settings

### PR DESCRIPTION
**Pull request by KonstantinRyazantsev**

### CODE REVIEW

File: src/Lykke.Job.RabbitMqMonitoring/Settings/JobSettings/RabbitMqMonitoringSettings.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ﻿using System;
 
-namespace Lykke.Job.RabbitMqMonitoring.Core.Settings.JobSettings
+namespace Lykke.Job.RabbitMqMonitoring.Settings.JobSettings
 {
         public class RabbitMqMonitoringSettings
         {
             public DbSettings Db { get; set; }
-            public RabbitMqConnection[] RabbitMqConnections { get; set; }
+            public RabbitMqConnectionSettings[] RabbitMqConnections { get; set; }
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.RabbitMqMonitoring/Settings/JobSettings/RabbitMqMonitoringSettings.cs]: can be IReadOnlyCollection<>

File: src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs [hunk 2]
```diff
@@ -34,29 +35,60 @@ public class CheckRabbitMqHandler : TimerPeriod
...
+
+            if (queue.Messages >= maxMessagesCount)
+            {
+                var title = connectionSettings.Title ?? new Uri(connectionSettings.Url).Host;
+                
+                await _log.WriteMonitorAsync(
+                    title,
+                    string.Empty,
+                    string.Empty,
+                    $"Queue '{queue.Name}' contains {queue.Messages} messages");
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs:73]: Didn't you forget .count()?

**Review by andrey-anokhin — APPROVED** [review]

File: src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs [hunk 3]
```diff
@@ -34,29 +35,60 @@ public class CheckRabbitMqHandler : TimerPeriod
...
             var tasks = new List<Task>();
 
-            foreach (var rabbitMq in _rabbitMqConnections)
+            foreach (var connectionSettings in _rabbitMqConnectionSettings)
             {
-                tasks.Add(ProcessConnection(rabbitMq));
+                tasks.Add(ProcessConnectionAsync(connectionSettings));
             }
 
             await Task.WhenAll(tasks);
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs]: use _rabbitMqConnectionSettings.Select(ProcessConnectionAsync(connectionSettings)) save a little memory

File: src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs [hunk 4]
```diff
@@ -4,28 +4,29 @@
...
         private readonly IRabbitMqManagementService _rabbitMqManagementService;
-        private readonly RabbitMqConnection[] _rabbitMqConnections;
+        private readonly RabbitMqConnectionSettings[] _rabbitMqConnectionSettings;
         private readonly int _maxMessagesCount;
         private readonly ILog _log;
 
         public CheckRabbitMqHandler(
             IRabbitMqManagementService rabbitMqManagementService,
-            RabbitMqConnection[] rabbitMqConnections, 
+            RabbitMqConnectionSettings[] rabbitMqConnectionSettings,
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs]: Can be IReadOnly*

File: src/Lykke.Job.RabbitMqMonitoring/Program.cs [hunk 5]
```diff
@@ -7,15 +7,17 @@ namespace Lykke.Job.RabbitMqMonitoring
 {
     public class Program
     {
+        public static string EnvInfo => Environment.GetEnvironmentVariable("ENV_INFO");
+
         public static void Main(string[] args)
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.RabbitMqMonitoring/Program.cs]: use async Task Main - save one thread

**Review by MikhailGerasimov — APPROVED** [review]

**Review by samodovdi — APPROVED** [review]

File: src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs (hunk 2 above)
Comment (KonstantinRyazantsev) [reply src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs:73]: Nope. `Messages` is int. Name is defined by the RabbitMq management API contract

**Pull request merged**